actors/builtin/reward/reward_logic.go

```diff
@@ -0,0 +1,72 @@
+package reward
+
+import (
+	abi "github.com/filecoin-project/specs-actors/actors/abi"
+	big "github.com/filecoin-project/specs-actors/actors/abi/big"
+	builtin "github.com/filecoin-project/specs-actors/actors/builtin"
+)
+
+var BaselinePowerAt = func(epoch abi.ChainEpoch) abi.StoragePower {
+	return big.NewInt(1 << 40)
+}
+
+// Computes RewardTheta, result is in Q.128 format
```

Comment:
A little more explanation would be great: readers won't necessarily understand the design intent. Here, perhaps "theta is the virtual network time that elapses as the realized integral over time of committed power approaches the baseline integral"

Or something, you can probably do better 😁

Reply:
I was planning to add more comments and tests, my first priority was getting a working code.